.github/workflows/_linux_accelerate.yml

```diff
@@ -0,0 +1,156 @@
+name: Linux Accelerate Test
+
+on:
+  pull_request:
+    branches:
+      - main
+    paths:
+      - '.github/scripts/spec.py'
+      - '.github/actions/print-environment/action.yml'
+      - '.github/workflows/_linux_accelerate.yml'
+  workflow_dispatch:
+    inputs:
+      pytorch:
+        required: false
+        type: string
+        default: 'nightly'
+        description: Pytorch branch/commit
+      python:
+        required: false
+        type: string
+        default: '3.10'
+        description: Python version
+      runner:
+        required: true
+        type: string
+        default: 'linux.idc.xpu'
+        description: Runner label
+      nightly_whl:
+        required: false
+        type: string
+        default: ''
+        description: Pytorch nightly wheel version
+      accelerate:
+        required: false
+        type: string
+        default: 'v1.2.1'
+        description: Accelerate version
+
+permissions: read-all
+
+jobs:
+  Torch-XPU-Accelerate-Tests:
+    runs-on: ${{ inputs.runner != '' && inputs.runner || 'linux.idc.xpu' }}
+    env:
+      CONDA_ENV_NAME: 'huggingface_accelerate_test'
+      WORK_DIR: 'accelerate'
+      NEOReadDebugKeys: 0
+      DisableScratchPages: 0
+      accelerate: ${{ inputs.accelerate != '' && inputs.accelerate || 'v1.2.1' }}
+      python: ${{ inputs.python != '' && inputs.python || '3.10' }}
+      pytorch: ${{ inputs.pytorch != '' && inputs.pytorch || 'nightly' }}
+      PYTORCH_DEBUG_XPU_FALLBACK: 1
+      ZE_AFFINITY_MASK: 0
+      PARSE_JUNIT: ${{ github.workspace }}/torch-xpu-ops/.github/scripts/parse-junitxml.py
+    steps:
+      - name: Checkout torch-xpu-ops
+        uses: actions/checkout@v4
+        with:
+          path: torch-xpu-ops
+      - name: Checkout Accelerate
+        uses: actions/checkout@v4
+        with:
+          repository: huggingface/accelerate
+          ref: ${{ env.accelerate }}
+          path: accelerate
+      - name: Prepare Conda ENV
+        run: |
+          which conda && conda clean -ay
+          conda remove --all -y -n $CONDA_ENV_NAME || rm -rf $(dirname ${CONDA_EXE})/../envs/$CONDA_ENV_NAME
+          conda create -y -n $CONDA_ENV_NAME python=${{ env.python }}
+          source activate $CONDA_ENV_NAME
+          pip install junitparser
+      - name: Prepare Stock XPU Pytorch
+        run: |
+          source activate $CONDA_ENV_NAME
+          if [ -z "${{ inputs.nightly_whl }}" ]; then
```

Comment:
Do we plan this test only target for torch cpu nightly wheel? latest nightly or a specific nightly version?

Reply:
I suggest to target it for latest nightly triggered per schedule daily. @chuanqi129 : are you ok if I will drop input parameter to set nightly version?

Reply:
I dropped `inputs.nightly_whl` for now and just install latest nightly. Note that this parameter in any case would need a change since it controls only `torch` version and does not allow to set version for `torchvision` and `torchaudio`. So, if needed, we can add these options later on.